### PR TITLE
aalto/jupyterhub: Direct link to courses

### PR DESCRIPTION
**Pull request by rkdarst**
- Made possible by jupyterhub/jupyterhub#3013

### CODE REVIEW

File: aalto/jupyterhub.rst [hunk 1]
```diff
@@ -262,6 +262,12 @@ FAQ and bugs
   and eventually needs to be cleaned up - after the respective course
   is done.
 
+* **I don't see the assignments for my course**.  There are different
```
Comment (Sefriol) [inline aalto/jupyterhub.rst:265]: Maybe this should be little more precise on what to do. Either some direct instructions on how to change spawners (i.e "On the upper right corner, click "Control Panel" next to "Logout", you should arrive to new page with either one or two buttons. First click "Stop my Server", wait a little bit and click "Start My Server" and choose a different spawner")

Or just step by step guide with pictures:
image
image
etc.

**Comment by rkdarst** [comment]
good point - I added a more detailed description of changing the profile.  I didn't add screenshots yet, because classic notebooks and jupyterlab are different.  Sometime later, we should consider adding a bunch of screenshots (maybe once it is fully jupyterlab?).

**Comment by Sefriol** [comment]
> good point - I added a more detailed description of changing the profile. I didn't add screenshots yet, because classic notebooks and jupyterlab are different. Sometime later, we should consider adding a bunch of screenshots (maybe once it is fully jupyterlab?).

Yeah, was thinking the same. JupyterLab integration is probably the point when we should look into this.

**Comment by rkdarst** [comment]
thanks for the reviews!

**Pull request merged**